chapters/synchronization.txt

```diff
@@ -4236,6 +4252,52 @@ ifdef::VK_NV_mesh_shader[]
     pname:dstStageMask must: not contain
     ename:VK_PIPELINE_STAGE_TASK_SHADER_BIT_NV
 endif::VK_NV_mesh_shader[]
+  * Each element of pname:pMemoryBarriers must: not have any access flag
+    included in its pname:srcAccessMask member that is not supported by any
+    of the pipeline stages in pname:srcStageMask, as specified in the
+    <<synchronization-access-types-supported, table of supported access
+    types>>
+  * Each element of pname:pMemoryBarriers must: not have any access flag
+    included in its pname:dstAccessMask member that is not supported by any
+    of the pipeline stages in pname:dstStageMask, as specified in the
+    <<synchronization-access-types-supported, table of supported access
+    types>>
+  * For each element of pname:pBufferMemoryBarriers, if its
+    pname:srcQueueFamilyIndex and pname:dstQueueFamilyIndex
+    members describe a <<synchronization-queue-transfers-release, queue
+    family release operation>>, and its pname:buffer member was created
+    with a sharing mode of ename:VK_SHARING_MODE_EXCLUSIVE, then its
+    pname:dstAccessMask member must: only contain access flags that are
+    supported by any of the pipeline stages in pname:dstStageMask, as
+    specified in the <<synchronization-access-types-supported, table of
+    supported access types>>
+  * For each element of pname:pBufferMemoryBarriers, if its
+    pname:srcQueueFamilyIndex and pname:dstQueueFamilyIndex
+    members describe a <<synchronization-queue-transfers-acquire, queue
+    family acquire operation>>, and its pname:buffer member was created
+    with a sharing mode of ename:VK_SHARING_MODE_EXCLUSIVE, then its
```

Comment:
This might be technically incorrect, as one of the extensions allow Acquire Barrier even for `VK_SHARING_MODE_CONCURRENT` (foreign queue).

If you intent do go with the link, I suggest you delete unnecessary information as mentioning the `*srcQueueFamilyIndex`. Otherwise it might as well be clearer to explicitly prescribe here what is Acquire Barrier, as I suggested previously.

Reply:
Eh, I can just remove the sharing mode exclusive part, as the VU for describing acquire/release barriers already bans cases where you'd specify an qfot when it doesn't make sense. Shall do that.